core-android/src/main/java/com/uber/sdk/android/core/auth/AuthUriAssembler.java

```diff
@@ -0,0 +1,51 @@
+package com.uber.sdk.android.core.auth;
+
+import android.net.Uri;
+import android.text.TextUtils;
+
+import androidx.annotation.NonNull;
+
+import com.uber.sdk.android.core.BuildConfig;
+import com.uber.sdk.android.core.utils.AppProtocol;
+import com.uber.sdk.core.client.SessionConfiguration;
+
+import java.util.Locale;
+
+public class AuthUriAssembler {
```

Comment:
FYI, these are in the SDK components, not sample app. Was that intended if this is to show an SDK less version?

Reply:
Yes, it was intended. Basically I want to put out 2 samples
1. applink usage with sdk
2. applink usage without sdk

AuthUriAssembler would stay inside the sdk and will be used with #1. 

For #2 I will create a new sample app, as you recommended, to keep things cleaner.